packages/govtnz-ds-website/src/pages/components/FileUpload.tsx

```diff
@@ -29,7 +29,7 @@ const PageContent = (props) => <React.Fragment><ExampleContainer>
         className: "example__iframe",
         src:"/components/FileUpload__example0.html",
         title:"Example: File upload",
-        height: 168
+        height: 165
```

Comment:
@Juliet-Brown Yes that's fine.

The examples are in an iframe, and rather than having a fixed height for every iframe the `yarn compile-markdown` script fires up Chrome for each example and measures the height of the content, and that's the number you see there.

I'm not sure why, but on subsequent runs of `yarn compile-markdown` the number can vary a bit, however changing from 168 to 165 pixels is harmless.